hlf/chaincode/draft.md

```diff
@@ -0,0 +1,69 @@
+# Hyperledger Draft Api
+
+In the following, the chaincode api for the "Draft"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## Transactions
+
+### AddOrExtendSignature
+- ID = addOrExtendSignature
+- Send
+    - hash(transactionName, params) :: String

```

Comment:
I'd just pass the transactionName and params to the transaction. This way we can do all hashing in chaincode and do not have to worry about that anywhere else.